internal/operators/ocs/validation_test.go

```diff
@@ -505,7 +505,7 @@ var _ = Describe("Ocs Operator use-cases", func() {
 				clust.AllHostsAreReadyToInstall:           {status: clust.ValidationSuccess, messagePattern: "All hosts in the cluster are ready to install"},
 				clust.IsDNSDomainDefined:                  {status: clust.ValidationSuccess, messagePattern: "The base domain is defined"},
 				clust.IsPullSecretSet:                     {status: clust.ValidationSuccess, messagePattern: "The pull secret is set"},
-				clust.SufficientMastersCount:              {status: clust.ValidationFailure, messagePattern: "Clusters with less than 3 dedicated masters or a single worker are not supported. Please either add hosts, or disable the worker host"},
+				clust.SufficientMastersCount:              {status: clust.ValidationFailure, messagePattern: "Clusters can only have 3 dedicated masters. Please either add hosts, or disable the worker host"},
```

Comment:
find a way to include a suitable message to single-node
Maybe we can make it depend on the availability_mode parameter, as single-node is with 'None' and regular clusters are with 'Full'

Reply:
There is a single node message [here](https://github.com/openshift/assisted-service/blob/913aba63aae8923400e8ff4e040c8cfc0d59897e/internal/cluster/validator.go#L369)